.github/workflows/build-client-packages.yml

```diff
@@ -54,6 +57,13 @@ jobs:
           fpm_args: "etc"
           fpm_opts: "--debug -n cvmfs-config-eessi -v ${{ steps.get_version.outputs.version }} -t deb -a all -s dir -C ./package --description 'CVMFS config repository package for EESSI.'"
 
+      - name: Build MacOS package
+        id: build-osxpkg
+        uses: bpicode/github-action-fpm@master
+        with:
+          fpm_args: "etc"
+          fpm_opts: "--debug -n cvmfs-config-eessi -v ${{ steps.get_version.outputs.version }} -t osxpkg -a all -s dir -C ./package --description 'CVMFS config repository package for EESSI.'"
```

Comment:
We could consider using Travis for this, they have Mac CI support...

Travis isn't very interesting these days though, only a very limited amount of credits available (although we don't need much in this case)

Reply:
Or a separate Github Action? They seem to support it as well:
https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
Ah, that's new...

Doesn't need to be a separate action, you can just run the same one on different platforms (linux/macos), and use conditionals to opt-in/out of steps, see https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif

Reply:
Seems to work:
https://github.com/bedroge/filesystem-layer/runs/1497929661?check_suite_focus=true

Reply:
Neat! Although this warning... https://github.com/bedroge/filesystem-layer/runs/1497929661?check_suite_focus=true#step:5:4 It won't ever go away, will it?

Reply:
You can use `pip3 install ansible` instead.

Reply:
Yep, it's gone now.